content/soda-gettingstarted/ak-sk-using-api.md

```diff
@@ -0,0 +1,159 @@
+---
```

Comment:
Should we move it to User Guide instead of Getting started

Reply:
Sure , will move.

Reply:
![image](https://user-images.githubusercontent.com/65751939/123737079-f96dae00-d8bf-11eb-95ba-de64557ad912.png)